[MI-2356] Updated docs (#33)
Brightscout/mattermost-plugin-servicenow-virtual-agent: docs/developer_docs.md

    - [Set up trusted media domains](https://docs.servicenow.com/bundle/quebec-now-intelligence/page/administer/virtual-agent/task/ccif-secure-file-upload.html)
      - [Configuring the form layout](https://docs.servicenow.com/en-US/bundle/sandiego-platform-administration/page/administer/form-administration/concept/configure-form-layout.html)

- **Invalid response**

>     We get an "undefined" response from Virtual Agent API in a specific case. You can read [this](https://www.servicenow.com/community/virtual-agent-nlu-forum/getting-improper-response-from-virtual-agent-bot-integration-api/m-p/255032) for more information.

Reviewer: May be this is better?
```suggestion
 If we get an "undefined" response from Virtual Agent API in some specific cases. Please refer to [this](https://www.servicenow.com/community/virtual-agent-nlu-forum/getting-improper-response-from-virtual-agent-bot-integration-api/m-p/255032) for more information.
```

Author: Changed the second line. But I think the first line if correct as we are telling a developer about the issue.

>     We get an "undefined" response from Virtual Agent API in a specific case. Please refer to [this](https://www.servicenow.com/community/virtual-agent-nlu-forum/getting-improper-response-from-virtual-agent-bot-integration-api/m-p/255032) for more information.
> 
>     [va.bot.to.bot.take.control_times](https://docs.servicenow.com/bundle/sandiego-servicenow-platform/page/administer/virtual-agent/concept/bot2bot.html)

Reviewer 2: ```suggestion
    We get an "undefined" response from Virtual Agent API in a specific case. Please refer to [this](https://www.servicenow.com/community/virtual-agent-nlu-forum/getting-improper-response-from-virtual-agent-bot-integration-api/m-p/255032) community question for more information about the issue. To bypass this issue, you should increase the value of `va.bot.to.bot.take.control_times` variable. You can read more about it [here](https://docs.servicenow.com/bundle/sandiego-servicenow-platform/page/administer/virtual-agent/concept/bot2bot.html).
```
